spec/components/validated_field_component_spec.rb

```diff
@@ -28,7 +28,10 @@
   it 'renders aria-describedby to establish connection between input and error message' do
     field = rendered.at_css('input')
 
-    expect(field.attr('aria-describedby')).to start_with('validated-field-error-')
+    expect(field.attr('aria-describedby').split(' ')).to include([
+      start_with('validated-field-hint-'),
+      start_with('validated-field-error-'),
+    ])
```

Comment:
Linter didn't like this, but it turns out the `[` are optional, which should make for an easy fix.

```suggestion
    expect(field.attr('aria-describedby').split(' ')).to include(
      start_with('validated-field-hint-'),
      start_with('validated-field-error-'),
    )
```